### PR TITLE
--underscore to create migrations with 'created_at' and 'updated_at'

### PR DESCRIPTION
**Pull request by Americas**
Needed a break from thinking, this feature was simple to do, so I just did it.

### CODE REVIEW

**Comment by sdepold** [comment]
Thanks :) Do you think you want to add tests?

**Comment by Americas** [comment]
added it for model:create, I can add it for migration:create, but it will follow the same paths. Will add it tomorrow.

File: test/model/create.test.js [hunk 1]
```diff
@@ -128,47 +128,63 @@ var _         = require('lodash');
...
-                  '        type: Sequelize.DATE',
-                  '      }'
-                ].join('\n')))
-                .pipe(helpers.ensureContent('});'))
-                .pipe(helpers.ensureContent('.dropTable(\'Users\')'))
-                .pipe(helpers.teardown(done));
+          ([
+            { underscored: true, createdAt: 'created_at', updatedAt: 'updated_at'},
+            { underscored: false, createdAt: 'createdAt', updatedAt: 'updatedAt'}
+          ]).forEach(function (attrUnd) {
```
Comment (sdepold) [inline test/model/create.test.js]: What is `attrUnd`? :D

Comment (Americas) [inline test/model/create.test.js]: its the ATTRibute for UNDerscored, you already have an attributes declared above, so I had to use something else

File: lib/tasks/migration.js [hunk 2]
```diff
@@ -12,7 +12,8 @@ module.exports = {
       'short': 'Generates a new migration file.',
       options: {
         '--name': 'Defines the name of the migration. ' +
-          clc.blueBright('Default: unnamed-migration')
+          clc.blueBright('Default: unnamed-migration'),
+        '--underscored': 'Set timestamps to snake_case'
```
Comment (sdepold) [inline lib/tasks/migration.js]: maybe "Use snake case for the timestamp's attribute names" ?

**Comment by sdepold** [comment]
test looks good

File: lib/tasks/migration.js (hunk 2 above)
Comment (Americas) [inline lib/tasks/migration.js]: Will change it when I get to my PC :wink:

**Comment by puzanov** [comment]
Who can merge? We need this very much

**Comment by sdepold** [comment]
@puzanov did you try out the branch? is it working properly?

**Comment by sdepold** [comment]
works nicely

**Comment by sdepold** [comment]
Released as 2.1.0

**Pull request merged**